### ISSUE TITLE
Learn rate on graph is offset

### ISSUE THREAD

**Issue by Xyem** [Zyin055/Inspect-Embedding-Training#19]
The vertical lines and labels for learning rates are offset (very obvious with cyclical learn profiles). Perhaps just a fence post error?

In the attachment, the high learn rate (0.05) is shown as being a small block at the start of a low learn (0.0005) section immediately after the "busy" section, instead of surrounding the busy section like it should be.

![vd-596-vector](https://user-images.githubusercontent.com/2192097/222451471-ba944e75-a9a2-4e31-87f4-9c65c6fba521.jpg)

**Comment by Zyin055** [comment]
This visual issue happens because your embedding files and loss rate aren't being saved to disk at the same rate. Likely you have your embeddings saved every X steps, but loss rate saved every Y steps (the learn rate is extracted from the loss rate .csv file).

**Comment by Xyem** [comment]
Ah, I see. Indeed, setting the CSV to be saved every step makes the lines render in the correct place. Thanks for explaining!
